src/routes/solid-router/reference/components/hash-router.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: HashRouter
+---
+
+The `HashRouter` is a top level component that manages the routing of your application.
+It's a client side router, utilizing hash-values in the URL, allow your single page application to simulate a multi-page application.
+
+The advantage of hash-routing is that the whole application can run from a single HTML file, which can be hosted on a static file server.
```

Comment:
```suggestion
Since hash-routing provides a way for an application to run from a single HTML file, it can be used when hosting on a static file server.
```